<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(176, 188, 157);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="profileF">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>40</y>
      <width>721</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="avatarL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>170</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(125, 125, 125,20);
border-radius: 85px;
border:1px;
border: 1px solid rgb(176, 188, 157);</string>
     </property>
     <property name="text">
      <string>Not set</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="avatarPB">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 188, 157);
border-radius:10px;
color:rgb(100,100,100);</string>
     </property>
     <property name="text">
      <string>Change Avatar</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="profileGB">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>451</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(85, 255, 255);
color: rgb(156, 156, 156);
border: 1px solid rgb(156, 156, 156);
</string>
     </property>
     <property name="title">
      <string>Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="nameLE">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>40</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:1px;
border: 1px solid rgb(176, 188, 157);
padding-left:5px;
color:rgb(80,80,80);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="placeholderText">
       <string>first name last name</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameL">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:rgb(20,20,20);</string>
      </property>
      <property name="text">
       <string>Name :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="infoL">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:rgb(20,20,20);</string>
      </property>
      <property name="text">
       <string>Info :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="infoTE">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>89</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:1px;
border: 1px solid rgb(176, 188, 157);
color:rgb(80,80,80);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>users currebt info will be written here and the word wrap is on so the text will  be readable. </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="privateDataF">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>721</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="privateDataGB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>661</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(85, 255, 255);
color: rgb(156, 156, 156);
border: 1px solid rgb(156, 156, 156);
</string>
     </property>
     <property name="title">
      <string>Private data</string>
     </property>
     <widget class="QLineEdit" name="emailLE">
      <property name="geometry">
       <rect>
        <x>162</x>
        <y>120</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:1px;
border: 1px solid rgb(176, 188, 157);
padding-left:5px;
color:rgb(80,80,80);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>19</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>userEmail@email.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="emailL">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:rgb(20,20,20);</string>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLE">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:1px;
border: 1px solid rgb(176, 188, 157);
padding-left:5px;
color:rgb(80,80,80);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>new password</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nameL_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:rgb(20,20,20);</string>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_repeatLE">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:1px;
border: 1px solid rgb(176, 188, 157);
padding-left:5px;
color:rgb(80,80,80);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>repeat new password</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="savePB">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 198, 157);
border-radius:10px;
color:rgb(100,100,100);</string>
     </property>
     <property name="text">
      <string>Save changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelPB">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>236</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(170, 170, 170,100);
border-radius:10px;
color:rgb(100,100,100);</string>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteAccountPB">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 62, 62);</string>
     </property>
     <property name="text">
      <string>Delete account</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
